com.microsoft.mrtk.spatialmanipulation/PlacementHub.cs

```diff
@@ -0,0 +1,374 @@
+            TransformFlags modifiedTransformFlags = TransformFlags.None;
+            SmoothTargetPose(ref targetTransform, ref modifiedTransformFlags);
+
+            if (rigidBody == null)
+            {
+                transform.SetPositionAndRotation(targetTransform.Position, targetTransform.Rotation);
+                transform.localScale = targetTransform.Scale;
+            }
+            else
+            {
+                // There is a Rigidbody. Potential different paths for near vs far manipulation
+                if (!useForces)
+                {
+                    rigidBody.MovePosition(targetTransform.Position);
+                    rigidBody.MoveRotation(targetTransform.Rotation);
+                }
+
+                transform.localScale = targetTransform.Scale;
+            }
+        }
+
+        #region smoothing
+        [SerializeField]
+        private bool useSmoothing = true;
+
+        public bool UseSmoothing
+        {
+            get { return useSmoothing; }
+            set { useSmoothing = value; }
+        }
+
+        [SerializeReference, InterfaceSelector(false)]
+        [DrawIf("shouldSmooth")]
+        private ITransformSmoothingLogic smoothingLogic = new DefaultTransformSmoothingLogic();
+
+        [SerializeField]
+        [Range(0, 1)]
+        [Tooltip("Enter amount representing amount of smoothing to apply to the movement. Smoothing of 0 means no smoothing. Max value means no change to value.")]
+        private float moveLerpTime = 0.001f;
+
+        /// <summary>
+        /// Enter amount representing amount of smoothing to apply to the movement. Smoothing of 0 means no smoothing. Max value means no change to value.
+        /// </summary>
+        public float MoveLerpTime
+        {
+            get => moveLerpTime;
+            set => moveLerpTime = value;
+        }
+
+        [SerializeField]
+        [Range(0, 1)]
+        [Tooltip("Enter amount representing amount of smoothing to apply to the rotation. Smoothing of 0 means no smoothing. Max value means no change to value.")]
+        private float rotateLerpTime = 0.001f;
+
+        /// <summary>
+        /// Enter amount representing amount of smoothing to apply to the rotation. Smoothing of 0 means no smoothing. Max value means no change to value.
+        /// </summary>
+        public float RotateLerpTime
+        {
+            get => rotateLerpTime;
+            set => rotateLerpTime = value;
+        }
+
+        [SerializeField]
+        [Range(0, 1)]
+        [Tooltip("Enter amount representing amount of smoothing to apply to the scale. Smoothing of 0 means no smoothing. Max value means no change to value.")]
+        private float scaleLerpTime = 0.001f;
+
+        /// <summary>
+        /// Enter amount representing amount of smoothing to apply to the scale. Smoothing of 0 means no smoothing. Max value means no change to value.
+        /// </summary>
+        public float ScaleLerpTime
```

Comment:
See comment about smoothing; looks like we have smoothing in two different places now

Reply:
Addressed in : https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11323#discussion_r1061900104